### ISSUE TITLE
Saved Events: ♡ icon doesn't have a text-to-speech label on the empty Saved events page

### ISSUE THREAD

**Issue by declanslevin** [redbadger/pride-london-app#213]
## Description of issue

When using a text-to-speech tool and tapping on the copy on the empty Saved events page it reads out:
iOS: "No favourites yet? Tap the _ on any events you like to save them here"
Android: "No favourites yet? Tap the _i_ on any events you like to save them here"

iOS just skips over the icon like it isn't there (hence the _), while Android reads it out as the letter i. Both will be confusing to the user. 

## Expected behaviour

When tapped the copy should be read out as:

"No favourites yet? Tap the _save this event button_ on any events you like to save them here"

## Steps to reproduce

- Make sure you have no saved events
- Enable the text-to-speech tool on your device
- Go to the Saved tab-
-Tap on the text "No favourites yet?..."

## OS / Browser / Device model

Galaxy S8
iPhone 7+

**Comment by declanslevin** [comment]
This has been fixed
